justfile

```diff
@@ -0,0 +1,20 @@
+
+#----------
+# Building
+#----------
+
+build: build-next build-0-6
+
+# Build the simulator
+build-next:
+    #!/usr/bin/env bash
+    pushd ./eg-next
+    cargo build --examples
+    popd
+
+# Run cargo test in release mode
+build-0-6:
+    #!/usr/bin/env bash
+    pushd ./eg-0.6
+    cargo build --examples
+    popd
```

Comment:
```suggestion
#----------
# Building
#----------

# Build the examples for all versions
build-all: (build "next") (build "0.6")

# Build the examples for the given version
build version:
    cd "eg-{{version}}" && cargo build --examples
```

The CI config will also need to be updated to call `build-all` if you decide to apply this suggestion.